src/Section/Section.js

```diff
@@ -0,0 +1,137 @@
+          <Card
+            style={cardInfo[1].imageSource !== undefined ? style : logoStyle}
+            image={cardInfo[1].imageSource}
+            createdAt={timeDiffChecker(timeDiffs[1])}
+            description={cardInfo[1].description}
+            uploadDate={todayIs()}
+          />
+        </a>
+        <a rel="noreferrer" href={cardInfo[2].url} target={"_blank"}>
+          <Card
+            style={cardInfo[2].imageSource !== undefined ? style : logoStyle}
+            image={cardInfo[2].imageSource}
+            createdAt={timeDiffChecker(timeDiffs[2])}
+            description={cardInfo[2].description}
+            uploadDate={todayIs()}
+          />
+        </a>
+        <a rel="noreferrer" href={cardInfo[3].url} target={"_blank"}>
+          <Card
+            style={cardInfo[3].imageSource !== undefined ? style : logoStyle}
+            image={cardInfo[3].imageSource}
+            createdAt={timeDiffChecker(timeDiffs[3])}
+            description={cardInfo[3].description}
+            uploadDate={todayIs()}
+          />
+        </a>
+        <a rel="noreferrer" href={cardInfo[4].url} target={"_blank"}>
+          <Card
+            style={cardInfo[4].imageSource !== undefined ? style : logoStyle}
+            image={cardInfo[4].imageSource}
+            createdAt={timeDiffChecker(timeDiffs[4])}
+            description={cardInfo[4].description}
+            uploadDate={todayIs()}
+          />
+        </a>
+        <a rel="noreferrer" href={cardInfo[5].url} target={"_blank"}>
+          <Card
+            style={cardInfo[5].imageSource !== undefined ? style : logoStyle}
+            image={cardInfo[5].imageSource}
+            createdAt={timeDiffChecker(timeDiffs[5])}
+            description={cardInfo[5].description}
+            uploadDate={todayIs()}
+          />
+        </a>
+        <a rel="noreferrer" href={cardInfo[6].url} target={"_blank"}>
+          <Card
+            style={cardInfo[6].imageSource !== undefined ? style : logoStyle}
+            image={cardInfo[6].imageSource}
+            createdAt={timeDiffChecker(timeDiffs[6])}
+            description={cardInfo[6].description}
+            uploadDate={todayIs()}
+          />
+        </a>
+        <a rel="noreferrer" href={cardInfo[7].url} target={"_blank"}>
+          <Card
+            style={cardInfo[7].imageSource !== undefined ? style : logoStyle}
+            image={cardInfo[7].imageSource}
+            createdAt={timeDiffChecker(timeDiffs[7])}
+            description={cardInfo[7].description}
+            uploadDate={todayIs()}
+          />
+        </a>
+        <a rel="noreferrer" href={cardInfo[8].url} target={"_blank"}>
+          <Card
+            style={cardInfo[8].imageSource !== undefined ? style : logoStyle}
+            image={cardInfo[8].imageSource}
+            createdAt={timeDiffChecker(timeDiffs[8])}
+            description={cardInfo[8].description}
+            uploadDate={todayIs()}
+          />
+        </a>
```

Comment:
[중요도: 높음]이렇게 API를 통해서 전달해준 데이터를 전부 뿌려주려면 Array.map이라는 함수를 사용하셔야 합니다.

```suggestion
       {cardInfo.map((cardData, index) => (
         <a rel="noreferrer" href={cardData.url} target={"_blank"} key={index}>
            <Card
              style={cardData.imageSource !== undefined ? style : logoStyle}
              image={cardData.imageSource}
              createdAt={timeDiffChecker(timeDiffs[index])}
              description={cardData.description}
              uploadDate={todayIs()}
            />
        </a>
       )}
```
형식으로 쓰면 좋습니다.

map을 사용할 때 가장 상위 element에 key를 사용하는 이유와 key의 값에는 단순히 숫자(index)가 들어가면 안좋습니다. 그 이유를 찾아보시면 더 도움이 될 것 같습니다!
제가 index를 쓴건 데이터가 정확하게 어떻게 오는지 몰라서 우선 이렇게 사용했습니다 ㅎㅎ